src/cider/exec/nextgen/jitlib/llvmjit/LLVMJITEngine.cpp

```diff
@@ -106,20 +109,40 @@ static llvm::SubtargetFeatures buildTargetFeatures(const CompilationOptions& co)
   return features;
 }
 
-llvm::TargetMachine* LLVMJITEngineBuilder::buildTargetMachine(
-    const CompilationOptions& co) {
+llvm::TargetMachine* LLVMJITEngineBuilder::buildTargetMachine() {
   return host_target->createTargetMachine(process_triple,
                                           process_name,
-                                          buildTargetFeatures(co).getString(),
+                                          buildTargetFeatures(module_.co_).getString(),
                                           buildTargetOptions(),
                                           llvm::None,
                                           llvm::None,
-                                          co.aggressive_jit_compile
+                                          module_.co_.aggressive_jit_compile
                                               ? llvm::CodeGenOpt::Aggressive
                                               : llvm::CodeGenOpt::Default,
                                           true);
 }
 
+void LLVMJITEngineBuilder::dumpASM(LLVMJITEngine& engine) {
+  const std::string fname = llvm_module_->getModuleIdentifier() + ".s";
```

Comment:
maybe we need to add some random to support dump code many times.

Reply:
right, maybe we use timestamp as suffix later.